src/components/menu/__internal__/submenu/submenu.component.js

```diff
@@ -185,6 +187,8 @@ const Submenu = React.forwardRef(
         onMouseOver={() => setSubmenuOpen(true)}
         onMouseLeave={() => closeSubmenu()}
         ref={submenuRef}
+        ariaLabel={ariaLabel}
```

Comment:
Should we be adding the same `ariaLabel` to two components?

Reply:
This is supplied for the case of menu items with only icons that have no supporting text